castle/cms/files/aws.py

```diff
@@ -1,13 +1,12 @@
 # for compat with python3, specifically the urllib.parse includes
 # noqa because these need to precede other imports
 from future.standard_library import install_aliases
-install_aliases()  # noqa
 
 import logging
 from datetime import datetime
 import StringIO
 from time import time
-from urllib.parse import urlsplit, urlunsplit, quote, quote_plus
+import six.moves.urllib as urllib
```

Comment:
why the changes to the lib and imports here in this file? how does this relate to the GA4 update?